Source/LoadByOS/WindowsConfigApp/FolderIcon.cs

```diff
@@ -0,0 +1,106 @@
+﻿using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Formats.Png;
```

Comment:
What is this library doing? I don't see any unfamiliar code in here.

Reply:
It's the image library that replaces system.drawing. it reads the image and writes it to a png. It's also needed in avalonia's captcha dialogue

Reply:
Yeah, I get that. I just don't see where it's used in here. ie -- I'm obviously missing something

Reply:
That's exactly what I was missing. It would have been easy to find in VS. It's so much easier to review PRs on github though so thanks for indulgine me.